app/controllers/users_controller.rb

```diff
@@ -0,0 +1,26 @@
+class UsersController < ApplicationController
+  def show
+    @user = User.find_by id: params[:id]
```

Comment:
nếu user bị nil mà gọi màn hình show cũng tèo, nên phải thêm code xử lý nil nhé 

```suggestion
    @user = User.find_by id: params[:id]
    return if @user
    
    flash[:warning] = "user not found..." # i18n nhé
    redirect_to root_path
```